<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chess Geeks - Main</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.432, y1:0.409091, x2:1, y2:1, stop:0 rgba(255, 255, 127, 243), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font:38pt &quot;Cooper Black&quot;;</string>
   </property>
   <property name="text">
    <string>Chess Geeks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScoreboard">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>182</y>
     <width>189</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Scoreboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlay">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>128</y>
     <width>189</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTheme">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>236</y>
     <width>189</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>189</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Times New Roman&quot;;
background-color:rgb(170, 0, 0);
color: rgb(255, 255, 127);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>120</y>
     <width>211</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font:38pt &quot;Cooper Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/king.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>321</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/logo2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>btnExit</zorder>
  <zorder>btnTheme</zorder>
  <zorder>btnPlay</zorder>
  <zorder>btnScoreboard</zorder>
 </widget>
 <resources>
  <include location="Chess resources.qrc"/>
 </resources>
 <connections/>
</ui>
